Wrappers/Python/cil/plugins/astra/__init__.py

```diff
@@ -1,2 +1,17 @@
+# -*- coding: utf-8 -*-
+#  Copyright 2018 - 2022 United Kingdom Research and Innovation
```

Comment:
is it correct to have the full year range here?

Reply:
as we are licensing it as apache in 2022

Reply:
but the copyright is from 2018-2022